.circleci/config.yml

```diff
@@ -63,6 +63,42 @@ jobs:
           paths:
             - data
             - cache
+  api:
```

Comment:
```suggestion
  recommendations:
```

unsure

Reply:
I don't think it's necessary. I agree we only have recommendations for now but it's likely that there will be multiple scripts in there later, all together building an API